tests/e2e/lib/setup-env.js

```diff
@@ -18,28 +17,36 @@ import {
 	loginToWpSite,
 } from './flows/jetpack-connect';
 import TunnelManager from './tunnel-manager';
+import { saveVideo } from 'playwright-video';
+import config from 'config';
+import path from 'path';
 
-const { PUPPETEER_TIMEOUT, E2E_DEBUG, CI, E2E_LOG_HTML } = process.env;
+const { TIMEOUT, E2E_DEBUG, CI, E2E_LOG_HTML } = process.env;
 let currentBlock;
 
 const defaultErrorHandler = async ( error, name ) => {
+	let filePath;
+
+	try {
+		filePath = await takeScreenshot( currentBlock, name );
```

Comment:
Why do we need to take screenshots on failures locally?

Reply:
Why not? The reports with screenshots are very helpful in debugging issues.

Reply:
I guess it depends on how one would debug issues locally. 

I prefer running E2E tests in headful mode with `E2E_DEBUG=true`, which will stop the execution on failure, timeout, or exception, which is quite handy, so I can poke around and see what's happened.